d2bs/kolbot/libs/bots/TeamDiablo.js

```diff
@@ -0,0 +1,462 @@
+                    }
+
+                    return 1;
+                }
+
+                return getDistance(me, a) - getDistance(me, b);
+            },
+            openSeal: function (classid) {
+                var i, seal;
+
+                for (i = 0; i < 5; i += 1) {
+                    // Try to get seal, if we fail we might be on a distance
+                    seal = getUnit(2, classid);
+                    if (!seal) {
+                        // Move closer to the seal
+                        Pather.moveToPreset(me.area, 2, classid, 2, 0, Team.communication.diablo.fast, true);
+                        seal = getUnit(2, classid);
+                    }
+
+                    // Now we really failed to get the seal
+                    if (!seal) {
+                        return false;
+                    }
+
+                    // Don't open seal if it is open already
+                    if (seal.mode) {
+                        return true;
+                    }
+
+                    // It appears the seal isn't opened yet. Open it
+                    Pather.moveToPreset(me.area, 2, classid, 2, 0);
+                    seal.interact();
+                    delay(500);
+
+                    if (!seal.mode) {
+                        if (classid === Units.DiabloSealSeizActive && Attack.validSpot(seal.x + 15, seal.y)) { // de seis optimization
+                            Pather.moveTo(seal.x + 15, seal.y);
+                        } else {
+                            Pather.moveTo(seal.x - 5, seal.y - 5);
+                        }
+                    } else {
+                        return true;
+                    }
+                }
+                return false;
+            },
+            getLayout: function (seal, value) {
+                var sealPreset = getPresetUnit(108, 2, seal);
+
+                if (!seal) {
+                    throw new Error("Seal (" + seal + ") preset not found. Can't continue.");
+                }
+
+                if (sealPreset.roomy * 5 + sealPreset.y === value || sealPreset.roomx * 5 + sealPreset.x === value) {
+                    return 1;
+                }
+
+                return 2;
+            },
+            followPath: function (path) {
+
+                Pather.moveTo(path[0], path[1], 3, getDistance(me, path[0], path[1]) > 50);
+                Attack.clear(30, 0, false, diablo.util.sort);
+
+            },
+            preattack: function (who) {
+
+            },
+            preattackDiablo: function () {
+                var trapCheck,
+                    tick = getTickCount();
+
+                while (getTickCount() - tick < 30000) {
+                    if (getTickCount() - tick >= 8000) {
+                        switch (me.classid) {
+                            case 1: // Sorceress
+                                if ([56, 59, 64].indexOf(Config.AttackSkill[1]) > -1) {
+                                    if (me.getState(121)) {
```

Comment:
change logic to
```
if (!me.getState(121)) {
    Skill.cast(...);
    continue;
}
```

Reply:
Same answer as on the other comment. This entire function will be removed since this is an duplicate from old code of kolton